File: prueba/Cliente/Web/IntegradorRecaudo/src/app/data/schema/clienteDto.ts
export class ClienteDto {
  id?: number;
  nombre_admin?: string;
  celular?: string;
  email?: string;
  documento?: string;
  documento_facturacion?: string;
  tipo_factura?: string;
  tipo_moneda?: string;
  fecha_creacion?: Date;
  tipo_cliente?: number;
  nombre_java?: string;
  puerto_web?: number;
  puerto_movil?: number;
  nombre_web?: string;
  id_servidor?: number;
  servidor?: string;
  id_zona_horaria?: number;
  zona_horaria?: string;
  id_version?: number;
  version?: string;
  estado?: string;

  constructor(init?: Partial<ClienteDto>) {
    if (init) {
      Object.assign(this, init);
    }
  }
}

File: prueba/Servidor/Core/integradorrecaudo/src/main/java/integradorrecaudo/clientes/dto/ClienteDTO.java
package integradorrecaudo.clientes.dto;

import com.fasterxml.jackson.annotation.JsonProperty;


import java.util.Date;

public record ClienteDTO(
        Integer id,
        @JsonProperty("nombre_admin")
        String nombreAdmin,
        String celular, String email, String documento,
        @JsonProperty("documento_facturacion")
        String documentoFacturacion,
        @JsonProperty("tipo_factura")
        String tipoFactura,
        @JsonProperty("tipo_moneda")
        String tipoMoneda,
        @JsonProperty("fecha_creacion")
        Date fechaCreacion,
        @JsonProperty("tipo_cliente")
        Integer tipoCliente,
        @JsonProperty("nombre_java")
        String nombreJava,
        @JsonProperty("puerto_web")
        Integer puertoWeb,
        @JsonProperty("puerto_movil")
        Integer puertoMovil,
        @JsonProperty("nombre_web")
        String nombreWeb,
        boolean activo,
        String servidor,
        @JsonProperty("zona_horaria")
        String zonaHoraria,
        String version) {
}
